second_coursework/YOLODetection[] detections
string class_prediction
float32 class_confidence